org/wildfly/mod_cluster/capability.adoc

```diff
@@ -0,0 +1,19 @@
+NAME: org.wildfly.mod_cluster
```

Comment:
@rhusar if we have two capabilities, we've gone wrong. :) I think this is just a problem of different people not knowing what other people are doing.